include/dxc/DXIL/DxilConstants.h

```diff
@@ -1278,18 +1274,6 @@ enum class OpCodeClass : unsigned {
   WaveReadLaneAt,
   WaveReadLaneFirst,
 
-  // WaveMatrix
-  WaveMatrix_Accumulate,
-  WaveMatrix_Annotate,
-  WaveMatrix_Depth,
-  WaveMatrix_Fill,
-  WaveMatrix_LoadGroupShared,
-  WaveMatrix_LoadRawBuf,
-  WaveMatrix_Multiply,
-  WaveMatrix_ScalarOp,
-  WaveMatrix_StoreGroupShared,
-  WaveMatrix_StoreRawBuf,
-
   // Work Graph intrinsics
   FinishedCrossGroupSharing,
```

Comment:
Will these still have the right values?

Reply:
Yes. The removed operations are changed into Deprecated.

Reply:
Ah, so the numeric values of the OpCodeClass entries don't matter?  

(I was misreading the diff and thought these were in the OpCode enum)

Reply:
Yes. OpCodeClass in DXIL is mapped to the Op function name.
The enum value for OpCodeClass doesn't matter.